includes/admin/upgrades/upgrade-functions.php

```diff
@@ -2861,3 +2870,42 @@ function give_v220_delete_wp_session_data(){
 
 	$wpdb->query( "DELETE FROM $wpdb->options WHERE option_name LIKE '_wp_session_%'" );
 }
+
+
+/**
+ * Update donor meta
+ * Set "_give_anonymous_donor" meta key to "0" if not exist
+ *
+ *
+ * @since 2.2.4
+ */
+function give_v224_update_donor_meta_callback() {
```

Comment:
@ravinderk why is this necessary? Can't we just query donors that are anonymous without having to explicitly set this meta value for all others? I'm holding off release `2.2.4` until we can discuss.